### PR TITLE
Account settings multiple fixes (second round)

### PR DESCRIPTION
**Pull request by leomendoza123**

### CODE REVIEW

**Comment by leomendoza123** [comment]
[[QA] Unable to submit form to remove duplicate account with keyboard](https://trello.com/c/22UNy18L/7961-qa-unable-to-submit-form-to-remove-duplicate-account-with-keyboard)

**Comment by leomendoza123** [comment]
[[QA] 500 error response when removing duplicate record that doesn't exist](https://trello.com/c/8LgHh5Cj/7960-qa-500-error-response-when-removing-duplicate-record-that-doesnt-exist)

**Comment by leomendoza123** [comment]
[[QA] Small fixes for deactivate account section](https://trello.com/c/bXn7VyWY/7959-qa-small-fixes-for-deactivate-account-section)

**Comment by leomendoza123** [comment]
[[QA] Whitespace needed in modal displayed after user finishes removing a duplicate account](https://trello.com/c/auKwm7Yd/7958-qa-whitespace-needed-in-modal-displayed-after-user-finishes-removing-a-duplicate-account)

**Comment by leomendoza123** [comment]
[[QA] Fix subsection names in new account settings page](https://trello.com/c/f4p69bAk/7948-qa-fix-subsection-names-in-new-account-settings-page)

File: src/app/account-settings/components/settings-actions-duplicated/settings-actions-duplicated.component.ts [hunk 1]
```diff
@@ -93,8 +93,13 @@ export class SettingsActionsDuplicatedComponent implements OnInit {
             .afterClosed()
         })
       )
-      .subscribe(() => {
-        this.loading.next(false)
-      })
+      .subscribe(
```
Comment (DanielPalafox) [inline src/app/account-settings/components/settings-actions-duplicated/settings-actions-duplicated.component.ts:96]: do we need duplicated the `loading.next`

**Review by DanielPalafox — CHANGES_REQUESTED** [review]

File: src/app/sign-in/pages/sign-in/sign-in.component.ts [hunk 2]
```diff
@@ -52,6 +53,7 @@ export class SignInComponent implements OnInit {
   ) {}
 
   ngOnInit() {
+    this.formSignInComponent.onSubmit()
```
Comment (DanielPalafox) [inline src/app/sign-in/pages/sign-in/sign-in.component.ts:56]: is this correct @leomendoza123? Do we wanna to call the onsubmit method in the ngOnInit

**Review by DanielPalafox — CHANGES_REQUESTED** [review]

File: src/app/account-settings/components/settings-actions-duplicated/settings-actions-duplicated.component.ts (hunk 1 above)
Comment (leomendoza123) [reply src/app/account-settings/components/settings-actions-duplicated/settings-actions-duplicated.component.ts:96]: Yes! 
When the observer `complete` or `error` we need the loading to stop

Comment (leomendoza123) [reply src/app/account-settings/components/settings-actions-duplicated/settings-actions-duplicated.component.ts:96]: Great catch!

**Review by amontenegro — APPROVED** [review]

**Pull request merged**